system/lib/compiler-rt/lib/sanitizer_common/sanitizer_platform_limits_posix.h

```diff
@@ -14,11 +14,29 @@
 #ifndef SANITIZER_PLATFORM_LIMITS_POSIX_H
 #define SANITIZER_PLATFORM_LIMITS_POSIX_H
 
-#if SANITIZER_LINUX || SANITIZER_MAC || SANITIZER_EMSCRIPTEN
+#if SANITIZER_LINUX || SANITIZER_APPLE || SANITIZER_EMSCRIPTEN
 
 #include "sanitizer_internal_defs.h"
 #include "sanitizer_platform.h"
 
+#if SANITIZER_APPLE
+#include <sys/cdefs.h>
+#if !__DARWIN_ONLY_64_BIT_INO_T
+#define SANITIZER_HAS_STAT64 1
+#define SANITIZER_HAS_STATFS64 1
+#else
+#define SANITIZER_HAS_STAT64 0
+#define SANITIZER_HAS_STATFS64 0
+#endif
+#elif SANITIZER_EMSCRIPTEN
+#define SANITIZER_HAS_STAT64 0
+#define SANITIZER_HAS_STATFS64 0
```

Comment:
I think we do support these. Though maybe we don't want to use them here for some reason?

Reply:
By default it looks like we don't define those:

```
ache/sysroot/lib/wasm32-emscripten/libsanitizer_common_rt.a" for subsequent builds)
/usr/local/google/home/sbc/dev/wasm/emscripten/system/lib/compiler-rt/lib/sanitizer_common/sanitizer_platform_limits_posix.cpp:214:33: error: invalid application of 'sizeof' to an incomplete type 'struct statfs64'
  unsigned struct_statfs64_sz = sizeof(struct statfs64);
                                ^     ~~~~~~~~~~~~~~~~~
```

We could define _GNU_SOURCE or _LARGEFILE_SUPPORT but I don't think any of that is necessary.. we didn't previously have SANITIZER_INTERCEPT_STATFS64 or SANITIZER_INTERCEPT_STAT64 defined and none of our tests are failing.

Reply:
Interesting... maybe they're behind a flag or something then. Or maybe I'm misremembering...